kernel/cifsd/Makefile

```diff
@@ -5,9 +5,9 @@ PKG_RELEASE:=1
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL:=https://github.com/cifsd-team/cifsd.git
-PKG_SOURCE_DATE:=2019-11-27
-PKG_SOURCE_VERSION:=b8675c8ac144ece00f3e6bcc5436c8ace99e23e9
-PKG_MIRROR_HASH:=3d67af87f30d837f95510663efc42f1451651dc235987408924b56cb277fc8e8
+PKG_SOURCE_DATE:=2019-12-07
+PKG_SOURCE_VERSION:=76063c33abb2217293f756d08521245405f9af39
+PKG_MIRROR_HASH:=18df9f7acdfce07fdc3f9f51693a937029d47c3775edd21ea5b12b5d15ad8761
```

Comment:
Release version are available (2.0.4) and codeload can be used as well.

Reply:
I follow master, since updates are still frequent and i prefer git over codeload for tiny projects like this one.